webapp/views.py

```diff
@@ -81,11 +81,45 @@ def releasenotes_redirect():
 
     Old apache redirects: https://pastebin.canonical.com/p/3TXyyNkWkg/
     """
-
-    ver = flask.request.args.get("ver")
+    releaseName = {
```

Comment:
The old solution relied on redirects at e.g.:

https://wiki.ubuntu.com/19.04/ReleaseNotes

This seems to be working fine. Why do we need to do it here? This will just mean we need to maintain another list of release codenames when hopefully it's already handled wiki side?